app/routes/userManagement.py

```diff
@@ -1,12 +1,36 @@
 from flask_login import current_user
-from flask import render_template, Blueprint, redirect
+from flask import render_template, Blueprint, redirect, request, flash
+from app.models.users import Merchant
+from app import db
 
 userManagement_bp = Blueprint('userManagement', __name__, url_prefix='/userManagement')
 
 
 @userManagement_bp.route("/", methods=['GET', 'POST'])
 def userManagement():
     if current_user.is_authenticated:
-        return render_template("userManagement.html")
+        if current_user.type == 'admin':
+            merchants = Merchant.query.filter_by(status='active')
+        else:
+            redirect("dashboard.html")
+
+        return render_template("userManagement.html", merchants=merchants)
+    else:
+        return redirect('login')
```

Comment:
Il serais bien d'avoir un `flash` message pour avertir  le client de qui se passe. Genre `you are not connected. Please login `
```suggestion
        return redirect('login')
        flash("you are not connected. Please login")
```